src/dom.js

```diff
@@ -639,6 +639,16 @@ export function isJsonScriptTag(element) {
 }
 
 /**
+ * Whether the element is a script tag with application/json type.
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function isJsonLdScriptTag(element) {
+  return element.tagName == 'SCRIPT' &&
+            element.getAttribute('type').toUpperCase() == 'APPLICATION/LD+JSON';
```

Comment:
nit: indentation

Reply:
Copied the indentation from here, but I can update both.
https://github.com/newmuis/amphtml-story/blob/c780f029fe98da0c0b7497a351939e6423d07104/src/dom.js#L636